raiden/transfer/architecture.py

```diff
@@ -106,9 +116,17 @@ class Event:
       upper layer will use the events for.
     """
 
-    __slots__ = ()
+    pass
+
+
+@dataclass
+class TransferTask(State):
+    # TODO: When we turn these into dataclasses it would be a good time to move common attributes
+    # of all transfer tasks like the `token_network_identifier` into the common subclass
```

Comment:
why?

Reply:
I believe i kept the comment as-is from the old class but didn't write this so i am not sure i can answer your question.